### ISSUE TITLE
Provide instruction/mnemonic documentation

### ISSUE THREAD

**Issue by mborgerson** [angr/angr-management#259]
It would be helpful to be able to click on an instruction and get some popup/pane view showing detailed specs for exactly what that instruction is supposed to do.

**Comment by zardus** [comment]
I think one way to do this is to show the vex or pcode; those are already
pretty heavily explanatory

**Comment by mborgerson** [comment]
Definitely, and I'm working now on a feature to show IR now actually; but what I had in mind for this issue was to show more documentation about a particular instruction from the associated manual text. Ghidra/P-code can actually help provide some of this information: https://github.com/angr/pypcode/blob/master/pypcode/processors/6502/data/manuals/6502.idx
